docs/UserGuide.md

```diff
@@ -156,16 +156,65 @@ Format: `exit`
+
+Exports the current list as a CSV file that can be imported through ContacX or other contact software such as Google/Outlook
+
+Format: `exportcsv EXPORTTYPE`
+* Exports CSV as per the specified format.
+* If `EXPORTTYPE` is left empty, it will default to `EXPORTTYPE = 1` and export in custom ContacX format
+* `EXPORTTYPE` options are
+   * `EXPORTTYPE=1` for ContacX format
+   * `EXPORTTYPE=2` for Google Contacts format
+   * `EXPORTTYPE=3` for Microsoft Outlook format
+* File will be saved on the directory `[JAR file location]/data/addressbook.csv`
+
+Examples:
+* `exportcsv 2`: Exports the current address book as a CSV file ready to be uploaded onto Google Contacts 
 ### Editing the data file
 
-AddressBook data are saved as a JSON file `[JAR file location]/data/addressbook.json`. Advanced users are welcome to update data directly by editing that data file.
+ContacX data are saved as a JSON file `[JAR file location]/data/contacx.json`. Advanced users are welcome to update data directly by editing that data file.
 
 <div markdown="span" class="alert alert-warning">:exclamation: **Caution:**
-If your changes to the data file makes its format invalid, AddressBook will discard all data and start with an empty data file at the next run.
+If your changes to the data file makes its format invalid, ContacX will discard all data and start with an empty data file at the next run.
 </div>
 
+### Importing data
+
+In order to import data from external sources into ContacX, there are the following options:
+
+* Replacing `contacx.json` file with a valid ContacX formatted generated by another system with ContacX
+* Simple CSV import (GUI-based option)
+* Advanced CSV import (Command based)
+
+### Simple CSV Import
+
+Select the `Import` button under `File->Import` on the top bar.
+
+!['basic import UI'](images/BasicImportUI.png)
+
+Upon click, the default OS file picker should show up, with filters for only .csv and .json files to be accepted. Accessing import via the top menu will allow users to import their task and appointments list as JSON/CSV files that were generated by another instance of ContacX.
+
+### Advanced Data Import : `importcsv`
+
+Format: `importcsv n/COLUMNNUM p/COLUMN_PERSON e/COLUMN_EMAIL a/COLUMN_ADDRESS t/COLUMN_TAGS`
+
+* Each argument represents the corresponding attribute for a person. n = Name, p = Phone, e = Email, a = Address, t = Tagged
+* If any argument is omitted, it will follow the next available column based on the default sequence of
+  Name - Phone - Email - Address - Tagged
+* Tutorials will have templates for importing for the following contact csv formats:
+  * Google Contacts
+  * Outlook Contacts
+
+Examples:
+* `importCSV n/2 p/3 e/5 a/6 t/4`
+  * Reads from a CSV file treating the 2nd row as names, 3rd row as phone numbers, 5th row as email addresses, 4th row as tagged.
+* `importCSV n/2`
+  * Reads from CSV treating the 2nd row as names, 3rd row as phone numbers, 4th row as email address, 5th row as tagged.
```

Comment:
Command capitalization is not consistent. Please choose a single way of capitalizing `csv`

Reply:
thanks for the catch accidentlaly named this wrongly